### PR TITLE
Portfolio: finish mobile version

### PR DESCRIPTION
**Pull request by veronica365**
On the mobile version
- Add the works section
- Add about my self section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -32,37 +31,222 @@ <h4>My Logo</h4>
...
+            </div>
+            <section id="works">
+                <article class="media">
+                    <div class="media-img img-1"></div>
+                    <div class="media-body">
+                        <h4>Tonic</h4>
+                        <div class="role">
+                            <span>CANOPY</span>
+                            <i></i>
+                            <span>Backend Dev</span>
```
Comment (Ibizugbe) [inline index.html]: I think your project is look a little bit different from the design in some aspects, kindly check out the images below.

your app
figma-intervak

the design
yoyr-interval

kindly implement the changes as it is a requirement that you stick to the design😉

File: index.html [hunk 2]
```diff
@@ -32,37 +31,222 @@ <h4>My Logo</h4>
...
+                        </a>
+                    </div>
+                </div>
+                <div>
+                    <ul>
+                        <li class="categories open">
+                            <a href="#"><span>Languages</span><span><img src="./assets/img/arrow-down.png"
+                                        alt="more" /></span></a>
+                            <div class="item-list">
+                                <div class="item">
```
Comment (Ibizugbe) [inline index.html:209]: I think your about section looks a little bit different from the design, kindly see the images below🤗
Your page
your_about

the design
design_about

kindly make the appropriate adjustments, as it is a requirement that we stick to the design for this milestone🤗

File: index.html [hunk 3]
```diff
@@ -32,37 +31,222 @@ <h4>My Logo</h4>
...
+                            <li><a href="#"><img src="assets/img/linkedin.png" alt="linkedin" /></a></li>
                             <li><a href="#"><img src="assets/img/angies.png" alt="angies" /></a></li>
+                            <li><a href="#"><img src="assets/img/twitter.png" alt="twitter" /></a></li>
+                            <li><a href="#"><img src="assets/img/medium.png" alt="medium" /></a></li>
                         </ul>
                     </div>
+                    <div class="read-more">
+                        <a href="#">
+                            Get my resume
+                        </a>
```
Comment (Ibizugbe) [inline index.html:200]: I can see you haven't implemented the given states for your buttons, kindly see the images below and implement the given states for all your buttons.
btnstate

File: index.html [hunk 4]
```diff
@@ -32,37 +31,222 @@ <h4>My Logo</h4>
...
+                                some
+                                of my
+                                work and
+                                experience! If you like what you see and have a project you need coded, don’t hestiate
+                                to
+                                contact
+                                me.</p>
+                        </div>
+                        <div class="connect">
+                            <h5>LET'S CONNECT</h5>
```
Comment (Ibizugbe) [inline index.html]: I think you should kindly wrap this in a `<h2>` if you have to use an `h` tag. it is important you always note that an h2 falls under an h1, an h3 is a heading under an h2, etc always remember the hierarchy, this is semantically and aids user accessibility. 🤗

File: index.html [hunk 5]
```diff
@@ -32,37 +31,222 @@ <h4>My Logo</h4>
...
+                            <a href="#">
+                                See Project
+                            </a>
+                        </div>
+                    </div>
+                </article>
+                <article class="media">
+                    <div class="media-img img-3"></div>
+                    <div class="media-body">
+                        <h4>Tonic</h4>
```
Comment (Ibizugbe) [inline index.html]: Kindly follow the header hierarchy rule as explained above🤗

File: index.html [hunk 6]
```diff
@@ -32,37 +31,222 @@ <h4>My Logo</h4>
...
+                                I'm William
+                                Glad to see you!
+                            </h1>
+                        </div>
+                        <div class="description">
+                            <p>I'm a software developer! I can help you build a product , feature or website Look
+                                through
+                                some
+                                of my
+                                work and
```
Comment (Ibizugbe) [inline index.html]: It is a requirement that you make use of a CSS grid for your layout in this section, I can see you made use of flexbox which is cool. but this milestone requires you make use of grid to make your skills with CSS grid stronger🤗 kindly make the right changes🤗

File: index.html [hunk 7]
```diff
@@ -32,37 +31,222 @@ <h4>My Logo</h4>
...
+                                <span><img src="./assets/img/arrow-right.png" alt="more" /></span>
+                            </a>
+                        </li>
+                        <li class="categories">
+                            <a href="#"><span>Skills</span><span><img src="./assets/img/arrow-right.png"
+                                        alt="more" /></span></a>
+                        </li>
+                    </ul>
                 </div>
             </section>
```
Comment (Ibizugbe) [inline index.html]: I suggest you put this section inside the "works" section as it is a project requirement that you create just the work and about section in this milestone and wrap other contents inside them🤗🤗🤗

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @veronica365 👋, 

Good job so far!😉
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- linters are checked.
- pull request is descriptive.


## Required Changes ♻️
_please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by veronica365** [comment]
All Feedback implemented and designs matched

**Review by bienvenuushindi — APPROVED** [review]
Hi @veronica365    , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Charlie Brown

## Highlights
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Comment by veronica365** [comment]
Hey @bienvenuushindi
Thanks for the review

**Pull request merged**